<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationViewer</class>
 <widget class="QWidget" name="OperationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operation Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="ops_buttons">
     <item>
      <widget class="QPushButton" name="op_add">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/cart-plus.svg</normaloff>:/resources/material-design/svg/cart-plus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="op_edit">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="op_remove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/cart-off.svg</normaloff>:/resources/material-design/svg/cart-off.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_add_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/cart-plus.svg</normaloff>:/resources/material-design/svg/cart-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add operation</string>
   </property>
  </action>
  <action name="action_edit_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>Edit operation</string>
   </property>
  </action>
  <action name="action_remove_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/cart-off.svg</normaloff>:/resources/material-design/svg/cart-off.svg</iconset>
   </property>
   <property name="text">
    <string>Remove operation</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>op_add</tabstop>
  <tabstop>op_edit</tabstop>
  <tabstop>op_remove</tabstop>
 </tabstops>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
